src/concrete/ml/common/utils.py

```diff
@@ -583,3 +582,33 @@ def all_values_are_of_dtype(*values: Any, dtypes: Union[str, List[str]]) -> bool
         supported_dtypes[dtype] = supported_dtype
 
     return all(_is_of_dtype(value, supported_dtypes) for value in values)
+
+
+def check_isclose_and_shape(a, b, **kwargs):
```

Comment:
actually now that I've checked more carefully, I don't think this is needed : we only call `numpy.isclose` without a .all on a if statement a single time, and I think it's an error ! Basically, `isclose` returns an array, not a bool, so we should never have something like `if array:` !

Reply:
agreed to keep is_allclose